src/richie/apps/courses/templates/courses/cms/course_detail.html

```diff
@@ -317,62 +340,77 @@ <h2 class="course-detail__title">{% blocktrans context "course_detail__title" %}
             {% block runs %}
                 <div class="course-detail__aside" property="isAccessibleForFree" content="true">
                     {% with runs_dict=course.course_runs_dict %}
-                        {% block runs_open %}
-                            <div class="course-detail__row course-detail__runs course-detail__runs--open">
-                                <h2 class="course-detail__title">
-                                    {% blocktrans context "course_detail__title" %}Course runs{% endblocktrans %}
-                                    {% render_model_add course "" "" "get_admin_url_to_add_run" %}
-                                </h2>
-                                {% for run in runs_dict.0|add:runs_dict.1|add:runs_dict.2 %}
-                                    {% include "courses/cms/fragment_course_run.html" %}
-                                {% empty %}
-                                    <div class="course-detail__empty">{% trans "No open course runs" %}</div>
-                                {% endfor %}
-                            </div>
-                        {% endblock runs_open %}
+                        <h2 class="course-detail__title">
+                            {% if runs_dict.0|add:runs_dict.1|add:runs_dict.2|visible_on_course_page:request.toolbar.edit_mode_active|length == 1 %}
+                                {% blocktrans context "course_detail__title" %}Other course runs{% endblocktrans %}
+                            {% else %}
+                                {% blocktrans context "course_detail__title" %}Course runs{% endblocktrans %}
+                            {% endif %}
+                            {% render_model_add course "" "" "get_admin_url_to_add_run" %}
+                        </h2>
+
+                        {% block runs_open_multiple %}
```

Comment:
I think we should display a message "No other course runs" because currently when there is only one course run, the display is weird.

<img width="504" alt="image" src="https://user-images.githubusercontent.com/9265241/161253034-379b2741-e051-4aa8-9a52-0df311beebba.png">